src/components/Cart/CartList.js

```diff
@@ -1,22 +1,27 @@
 import CartItem from "./CartItem";
 import CommonContainer from "../Base/Containers/CommonContainer";
-import { useContext } from "react";
-import CartContext from "../../store/cart-context";
 import Button from "../Button/Button";
 import { Link } from "react-router-dom";
 import { List, TotalWrapper } from "./CartStyles";
 import { ROUTES } from "../../common/constants";
 import Notice from "../Notice/Notice";
+import { useSelector } from "react-redux";
 
 const CartList = () => {
-  const cartContext = useContext(CartContext);
+  const cartItems = useSelector((state) => {
+    return state.cart.items;
+  });
+
+  const cartTotalAmount = useSelector((state) => {
```

Comment:
it's better to keep your selector functions somewhere in another file and export it for convinience / potential reusability etc